src/parsers/manifest/dash/common/indexes/timeline/timeline_representation_index.ts

```diff
@@ -549,6 +555,9 @@ export default class TimelineRepresentationIndex implements IRepresentationIndex
     if (this._index.timeline === null) {
       this._index.timeline = this._getTimeline();
     }
+    if (!this._isDynamic) {
```

Comment:
There's no point in cleaning non-dynamic MPD.
Cleaning means removing old now-unreachable segments, but if the MPD never changes, the older segments should not become unreachable.

I disable that part to protect against future potential bug for VoD contens (those bugs would be limited only to live contents).